shared/cfg/guest-os/Linux/JeOS/27.cfg

```diff
@@ -0,0 +1,58 @@
+- 27:
+    variants:
+        - aarch64:
+            vm_arch_name = aarch64
+        - ppc64:
+            vm_arch_name = ppc64
+        - ppc64le:
+            vm_arch_name = ppc64le
+        - s390x:
+            vm_arch_name = s390x
+        - x86_64:
+            vm_arch_name = x86_64
+    image_name = images/jeos-27-${vm_arch_name}
+    os_variant = fedora27
+    boot_path = "images/pxeboot"
+    # Currently we only support unattended_install.url.extra_cdrom_ks
+    no unattended_install..floppy_ks, unattended_install..cdrom, svirt_install
+    unattended_install.url.extra_cdrom_ks:
+        # Use secondary as default url, because it's common to most variants
+        url = http://dl.fedoraproject.org/pub/fedora-secondary/releases/27/Server/${vm_arch_name}/os/
+        kernel = images/jeos-27-${vm_arch_name}/vmlinuz
+        initrd = images/jeos-27-${vm_arch_name}/initrd.img
+        # Unattended-file does not require any changes
+        unattended_file = unattended/JeOS-25.ks
```

Comment:
I was confused about the lacka of JeOS.27 kickstart files, but now I get the point.